modules/lists/cow_list.py

```diff
@@ -0,0 +1,22 @@
+
+
+class CowList:
+    def __init__(self, input_list):
+        self.backing_list = input_list
+        
+    def __getitem__(self, i):
+        return self.backing_list[i]   
+
+    def append(self, item):
+        modified_list = []
+        for i in self.backing_list:
```

Comment:
```suggestion
        modified_list = self.backing_list[:]
```
This does exactly what you’re doing, but more pythonically!